Sources/BraintreeVenmo/BTVenmoRequest.swift

```diff
@@ -49,7 +49,11 @@ import Foundation
     /// Whether the customer's shipping address should be collected and displayed on the Venmo paysheet.
     /// Defaults to `false`
     public var collectCustomerShippingAddress: Bool = false
-    
+
+    /// Indicates whether the purchase amount is the final amount.
```

Comment:
Nit - should we include that this will change the Venmo app UI?

Reply:
We can, do we want to add something like "Will update the Venmo UI to indicate the amount will not change"? We should probably avoid calling out specific language incase venmo changes the wording at some point. What do you think?

Reply:
Hm, up to you. Maybe something like `removes "subject to change" notice in Venmo app UI`?

Reply:
Done: https://github.com/braintree/braintree_ios/pull/1192/commits/9aa6ff958aaf28b3fc5d3a970f1dd80006f5801b